### PR TITLE
DOC: added explanation of each of the documentation sections

### PR DESCRIPTION
**Pull request by afeld**
This is meant to help contributors understand what information belongs where.

- [ ] ~~closes #xxxx~~
- [ ] tests ~~added /~~ passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] ~~whatsnew entry~~

### CODE REVIEW

File: doc/source/development/contributing_documentation.rst [hunk 1]
```diff
@@ -21,6 +21,14 @@ the next person.
 About the pandas documentation
 --------------------------------
 
+As you can see from the top navigation, the documentation is split into five sections:
+
+* **Getting Started:** This is where new pandas users should start. It explains how to get set up, and how to get oriented with pandas.
+* **User Guide:** These are deeper dives into specific pandas features, meant for users with a bit more experience.
```
Comment (datapythonista) [inline doc/source/development/contributing_documentation.rst:27]: I'm not sure  I agree the user guide is targeting users with more experience.

**Review by datapythonista — APPROVED** [review]
lgtm, seems like the CI is broken, we'll have to wait until it's fixed.

**Comment by datapythonista** [comment]
@afeld do you want to merge master here?

**Comment by simonjayhawkins** [comment]
> @afeld do you want to merge master here?

@afeld can you do this and respond to https://github.com/pandas-dev/pandas/pull/41689#discussion_r641739368

**Comment by mroeschke** [comment]
Thanks for the PR, but it appears that this PR has gone stale. Close due to inactivity but if you're interested in continuing let us know and we can reopen.

**Pull request closed without merging**